src/locales/en/mystery-encounter.ts

```diff
@@ -82,4 +82,25 @@ export const mysteryEncounter: SimpleTranslationEntries = {
   "fight_or_flight_option_3_selected": "You leave the strong Pokémon\nwith its prize and continue on.",
   // "fight_or_flight_outro_win": "The mysterious challengers were defeated!",
 
+  "sleeping_snorlax_intro_message": `As you walk down a narrow pathway, you see a towering silhouette blocking your path.
+  $You get closer to see a Snorlax sleeping peacefully.\nIt seems like there's no way around it.`,
+  "sleeping_snorlax_title": "Sleeping Snorlax",
+  "sleeping_snorlax_description": "Do you attack it to try and get it to move, or wait for it to wake up?",
+  "sleeping_snorlax_query": "What will you do?",
+  "sleeping_snorlax_option_1_label": "Fight it",
+  "sleeping_snorlax_option_1_tooltip": "(+) Fight Sleeping Snorlax",
+  "sleeping_snorlax_option_2_label": "Wait for it to move",
+  "sleeping_snorlax_option_2_tooltip": "(75%) Party member falls asleep\n(25%) All Pokémon healed",
+  "sleeping_snorlax_option_3_label": "Steal",
+  "sleeping_snorlax_option_3_tooltip": "(+) Leftovers",
```

Comment:
I think when you specify that they will get leftovers, you're taking away a cool moment for the player, where they go "omfg I stole leftovers? let's goooo!". So this goes hand in hand with my last comment. Unless it's gonna straight up end their run, they should not get much forewarning. Let them explore.

Reply:
I disagree here as well, but only because it's not clear that Steal is a "secret" option that only rarely shows up if you have a specific item stealing move on one of the pokemon. It's a rare option to have available to you, since knock off/ thief/ pluck etc. require specific circumstances or pokemon to get.

If you manage to trigger one of these secret events, you should be rewarded and know what the rewards are. Again, it let's you make meaningful decisions in future runs (Oh, if I hold on to Pluck in X biome, maybe it'll be worth it if I can get the Snorlax encounter) - as opposed to (Idk what the item I'll get here is, maybe it's a berry. I guess I'll just fight.)

Reply:
I think we agree on the fundamentals, then.

What about having the option 3 have a tooltip that says "Steal" but shows greyed out and is unclickable. Hovering over it without the requirement (let's say Thief for the sake of simplicity) says something like "An item-stealing item could help you here". If you have the move, the option appears white like any other and then yeah, it can fully display that you get Leftovers. I wouldn't mind it here because it just a different sense of wonder happening => first time around you didn't have one of said moves, so you go prepared for the 2nd. It's cool.

Reply:
I like that a lot, actually. It makes it really interesting to be like..
"hmm... I wonder why that's grayed out. Maybe if I...?"
That train of thought kinda gives us the best of both worlds.

Reply:
For option 2, I kind of prefer some language like:
(75%) Wait a long time
(25%) Wait a short time

For option 3 (and for all options with requirements): I really love the idea of disabling/greying out the option instead of fully hiding it. And giving the player a hint for how they might enable that option in the future as @InnocentGameDev suggested. When the option requirement is met for 3 specifically, I think (+) Steal Leftovers is fine